crates/fiber-lib/src/fiber/channel.rs

```diff
@@ -1686,18 +1687,20 @@ where
             }
         }
         state.retryable_task_last_run_at = Some(now_timestamp_as_millis_u64());
-        let mut pending_tlc_ops = state.tlc_state.get_pending_operations();
-        pending_tlc_ops.retain_mut(|retryable_operation| {
+        let mut check = async |retryable_operation: &mut RetryableTlcOperation| {
```

Comment:
This closure applies the tlc operations, so maybe `apply_tlc_op` is a better name.